src/main/java/reposense/commits/CommitResultAggregator.java

```diff
@@ -136,7 +136,7 @@ private static Date getStartOfDate(Date current) {
     }
 
     private static Date getStartDate(List<CommitResult> commitInfos) {
-        Date min = new GregorianCalendar(1970, 1, 1).getTime();
+        Date min = new GregorianCalendar(1970, Calendar.JANUARY, 1).getTime();
```

Comment:
Do also make the other `GregorianCalendar` implementation to use this method `Date date = new Date(Long.MIN_VALUE);`.

Reply:
Made the changes